samza-api/src/main/java/org/apache/samza/system/SystemAdmin.java

```diff
@@ -28,6 +28,17 @@
  * utility methods that Samza needs in order to interact with a system.
  */
 public interface SystemAdmin {
+
+  /*
+   * Connect to the underlying system and prepare to begin serving requests
```

Comment:
Update documentation. Don't need to mention connecting and serving requests, just say something generic like start the SystemAdmin. Same for stop.

Reply:
This documentation was based on the documentation of SystemConsumer.start(), which says "Tells the SystemConsumer to connect to the underlying system, and prepare to begin serving messages when poll is invoked".

Sure, I updated the documentation of start() to "Start up API for bringing up a single instance of SystemAdmin.". And I updated the documentation of stop() to "Shutdown API for shutting down a single instance of the SystemAdmin". Does this look good?

Reply:
Fixed as suggestd.